Makefile

```diff
@@ -32,7 +35,7 @@ $(SERVICES):
 $(DOCKERS):
 	$(call make_docker,$(@))
 
-dockers: $(DOCKERS)
+dockers: test $(DOCKERS)
```

Comment:
No need to test when you build Dockers IMHO. Test should be separate procedure, executed during development and with CI. Our CI already executes the tests, but now CI can be changed to call `make test`.

Reply:
I added them just to be sure that everything works before pushing to Docker Hub, but I will remove them